lab-justin/test/note-route-get-test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
+const port = process.env.PORT || 3000;
+const server = require('../server');
+const serverUrl = `http://localhost:${port}`;
+
+describe('testing note-route-two module', function(){
+
+  before(function(done) {
+    if(!server.isRunning){
+      server.listen(port, function(){
+        done();
+      });
+      return;
+    }
+    done();
+  });
+
+  after(function(done){
+    if (server.isRunning){
+      server.close(function(){
+        server.isRunning = false;
+        done();
+      });
+      return;
+    }
+    done();
+  });
+
+  describe( 'testing /api/note GET method', function(){
+    before((done) => {
+      request.post(`${serverUrl}/api/note`)
+        .send({content: 'test note !!!'})
+        .end((err, res) => {
+          request.get(`${serverUrl}/api/note?id=${res.body.id}`)
+          .end((err, res) => {
```

Comment:
You're not doing anything with err here. I would _always_ do a check to make sure that err is null.